### PR TITLE
Minor change to UG contacts management

### PR DESCRIPTION
**Pull request by dloh2236**
Formatting change, put delete command info in an admonition box
Several spelling errors
Update index.md to FreelanceBuddy

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -223,30 +228,37 @@ Deletes the specified contact from the **Contacts** tab using the keyword. Will
...
 > `Hans Bo` → 3. Hans Gruber
 >             4. Bo Yang 
+
 * The order of the keywords does not matter. e.g. Hans Bo will match Bo Hans
-> `Bo Hans` → 3. Hans Gruber
-> 
->             4. Bo Yang
+> `Bo Hans` → 3. Hans Gruber `/` 4. Bo Yang
+
 * Only the `NAME` of the contact is searched
```
Comment (sushiyade) [inline docs/UserGuide.md:250]: Is there a difference between this example and the one above it on lines 243 - 245? Is there a reason to have a '/' character in one and not the other?

File: docs/index.md [hunk 2]
```diff
@@ -3,17 +3,33 @@
...
+  one place, making it easy to access and update their
+  information whenever needed.
+
+- **Project Tracking**: Keep track of project completion
+  statuses, ensuring you're always aware of your ongoing work.
 
-* If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
-* If you are interested about developing AddressBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
+* If you are interested in using FreelanceBuddy, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
+* If you are interested about developing FreelanceBuddy, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
```
Comment (sushiyade) [inline docs/index.md:32]: Looks great after the readability and formatting changes!

**Review by sushiyade — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (dloh2236) [reply docs/UserGuide.md:250]: ok i have changed it to only show the example once and the next point to refer to the result above

**Review by flexibo — APPROVED** [review]
LGTM

**Review by sushiyade — APPROVED** [review]

**Pull request merged**